swift/ios-shoppe-demo/Views/CartTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CartTableViewController.swift
+//  ios-shoppe-demo
+//
+//  Created on 4/7/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CartTableViewController: UITableViewController {
+
+    var order: Order = Order()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        print("\(order.cartOrderTotal())")
+
+        navigationController?.navigationBar.tintColor = .white
+        tableView.register(UINib(nibName: "CartTableViewCell", bundle: nil), forCellReuseIdentifier: "cell")
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cell2")
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 3
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 3:
+            return order.items.count
+        default:
+            return 1
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch indexPath.section {
+        case 0:
+            let cell =  tableView.dequeueReusableCell(withIdentifier: "cell") as? CartTableViewCell
+
+            cell?.tableViewController = self
+            cell?.setupCartOrder()
+
+            return cell ?? UITableViewCell()
+        case 1:
+            let cell = UITableViewCell()
+
+            cell.textLabel?.font = UIFont(name: "Halvetica-Bold", size: 30)
+            cell.textLabel?.textAlignment = .left
+            cell.textLabel?.textColor = .systemIndigo
+
+            cell.textLabel?.text = "Products In Cart:"
+
+            return cell
+        default:
```

Comment:
The default case seems like it's the result of an if it's not 0 or 1 it must be 2.  If so, one suggestion might be to be more explicit and create a case 2 so it's really clear what that code block is intended to do.  The default case might then return a blank cell.  I see a lot of examples that use default as the fall through, but I was thinking that if you were to add another cell in the future, you'd have to add a new case to cover the old index and then update the code in default.  So rather than that, just use default as a no-op and have cases to cover all indexes.  The addition of a cell is then just the addition of a new case.  Just a thought ...